src/rdf4cpp/rdf/storage/node/reference_node_storage/SyncReferenceNodeStorageBackend.cpp

```diff
@@ -4,25 +4,26 @@
 namespace rdf4cpp::rdf::storage::node::reference_node_storage {
 
 SyncReferenceNodeStorageBackend::SyncReferenceNodeStorageBackend() noexcept {
```

Comment:
I tried that in the original PR that introduced `UnsyncReferenceNodeStorageBackend`, but I stopped at some point because I decided that the additional effort was not worth it